sway/commands/hide_edge_borders.c

```diff
@@ -32,14 +32,13 @@ struct cmd_results *cmd_hide_edge_borders(int argc, char **argv) {
 	} else if (strcmp(argv[0], "both") == 0) {
 		config->hide_edge_borders = E_BOTH;
 	} else if (strcmp(argv[0], "smart") == 0) {
-		config->hide_edge_borders = E_SMART;
+		config->hide_edge_borders_smart = E_SMART;
```

Comment:
I think this and `smart_no_gaps` should also set `config->hide_edge_borders = E_NONE` as well. It appears that `hide_edge_borders smart|smart_no_gaps` is the same behavior as `hide_edge_borders none; smart_borders on|no_gaps`.

Reply:
Makes sense. I put that equivalence in the doc.